releases/release-1.31/release-notes/maps/pr-124091-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 124091
+releasenote:
+  text: kubelet no longer crashes when a Dynamic Resource Allocation (DRA) driver
```

Comment:
```suggestion
  text: Fixed kubelet so it would no longer crash when a DRA(Dynamic Resource Allocation) driver returns a nil as part of the Node(Un)PrepareResources response instead of an empty struct (Did not affect drivers written in Go, first showed up with a driver written in Rust).
```